src/auth/auth.service.spec.ts

```diff
@@ -129,6 +156,63 @@ describe("AuthService", () => {
     });
   });
 
+  describe("user local login", () => {
+    it("should be login successfully.", async () => {
+      const mockUser = {
+        email: "test@example.com",
+        id: 1,
+      };
+      const expectedToken = "mocked_token";
+      const expectedStatusCode = HttpStatus.CREATED;
+
+      jest.spyOn(jwtService, "sign").mockReturnValue(expectedToken);
+
+      const result = await authService.login(mockUser);
+
+      expect(result).toEqual({
+        accessToken: expectedToken,
+        statusCode: expectedStatusCode,
+      });
+    });
+
+    it("should be validate successfully.", async () => {
+      const mockAccount = "test";
+      const mockPassword = "Password@123";
+      const mockUser: Partial<UserEntity> = {
+        account: "test",
+        email: "test@example.com",
+        id: 1,
+        name: "test",
+        password:
+          "$2b$05$zc4SaUDmE68OgrabgSoLX.CDMHZ8SD/aDeuJc7rxKmtqjP5WpH.Me",
+      };
+
+      jest
+        .spyOn(userService, "findOne")
+        .mockImplementation(async () => mockUser as UserEntity);
+
+      const result = await authService.validateUser(mockAccount, mockPassword);
+
+      expect(result).toBeDefined();
+      expect(result).toEqual({
+        email: mockUser.email,
+        id: mockUser.id,
+      });
+    });
+
+    it("should be validate failure.", async () => {
```

Comment:
lacks test for password not correct but the account exists

Reply:
Revised.
when the account does not exist should be validate failure.
when the account exist but password not correct should be validate failure.

[feat: add unit test for non-existent account and wrong password](https://github.com/Cophr/backend/pull/10/commits/e8557f8bb3c9a292e4a0b93182444efac0a3a4a3)